templates/boot/generic-barebox-tftp-ramdisk-boot-template.jinja2

```diff
@@ -0,0 +1,20 @@
+{% extends 'base/kernel-ci-base-tftp-deploy.jinja2' %}
+{% block metadata %}
+{{ super() }}
+{% endblock %}
+{% block main %}
+{{ super() }}
+{% endblock %}
+{% block actions %}
+{%- block deploy %}
+{{ super () }}
+{%- endblock %}
```

Comment:
same comment for all these blocks that are just using `(super)`

Reply:
Then you could change these in all other templates like uboot. They are pretty doing the same. It was basically copied from them.

Reply:
Oh, right, I had never noticed that.  So yes I guess the other templates should be cleaned up too.